contributor-docs/adrs/adr-008-experimental-components.md

```diff
@@ -0,0 +1,70 @@
+# ADR 008: Lowering the barrier to entry for experimental components
+
+## Status
+
+Proposed
+
+&nbsp;
+
+## Context
+
+#### Recap: Drafts
+
+As we work on new components (or rewrites of old components), we often need to start with a "prototype"[^1] and not include them in the semantically versioned (semver-ed) main bundle.
+
+The approach that has served us well since Dec 2021 has been "drafts" (along with causing a lot of confusion).
+
+[ADR-006](./adr-006-drafts.md) explains our use of drafts, but here are the relevant parts:
+
+- When we are not sure about the API of a component, we put it in a different bundle: "primer/react/drafts"
+- A draft has experimental API, but follows the development conventions of primer/react
+- Components in drafts are accompanied by a design spec and go through the process of accessibility reviews. A component can check multiple boxes in the component lifecycle.
+- The contract with consumers is that you are opting into a component in which we don't have 100% confidence yet.
+
+[^1]: Calling this a prototype and not experimental, because that already means something else in the maturity lifecycle.
+
+&nbsp;
+
+#### Problems with drafts
+
+1. **Incompatible with semver**: Because the expectation with drafts is that we are not 100% confident about the API or structure of a component, we slip in breaking changes to draft components as minor releases to primer/react. On the consumer side, this makes every upgrade, even a patch, a potentially breaking upgrade. So, you are effectively opting out of your semver.
+
+1. **Confusion about maturity**: Because of the generic name, "drafts" causes a lot of confusion, especially with "experimental". It is a prototype or experiment, but NOT in the way the [component lifecycle]([https://primer.style/contribute/component-lifecycle]) defines "Experimental" maturity.
```

Comment:
```suggestion
2. **Confusion about maturity**: Because of the generic name, "drafts" causes a lot of confusion, especially with "experimental". It is a prototype or experiment, but NOT in the way the [component lifecycle]([https://primer.style/contribute/component-lifecycle]) defines "Experimental" maturity.
```

I think the automatic numbering in this list is off.

Reply:
Letting [rendered markdown](https://github.com/primer/react/blob/adr-experimental-package/contributor-docs/adrs/adr-008-experimental-components.md#problems-with-drafts) take care of that :)